### ISSUE TITLE
Create organization for IndexData (consumer only)

### ISSUE THREAD

**Issue by corylown** [pod4lib/aggregator#548]
Blocked by #482

**Comment by JohnMarkOckerbloom** [comment]
Is this now unblocked, or are there still things blocking it?  (#482, cited above, has been closed.)

**Comment by corylown** [comment]
I've created an organization for IndexData, but I'm waiting for them to let me know who I should invite.

**Comment by corylown** [comment]
Closing. Created a new issue #651 to track getting list of people to invite from IndexData.
